public/help/collection-preferences.html

```diff
@@ -2,8 +2,8 @@ <h4>Collections, Challenges and Gifts Preferences</h4>
 
 <dl id="help">
   
-  <dt>Automatically agree to your work being collected by others in the Archive</dt>
-  <dd>By default, if another user adds your work to a collection, you will be asked to confirm whether you are willing for your work to be displayed as part of that collection. If you'd like people to be able to add your work to collections without waiting for confirmation from you, enable this option.</dd>
+  <dt>Allow others to invite my works to collections</dt>
+  <dd>By default, other users may not add your work to a collection. If you'd like people to be able to add your work to collections, enable this option. You will also need to approve invitations to a collection.</dd>
```

Comment:
@sarken the ticket mentioned potential new wording from the Docs team; I just made this up, so maybe someone from there would want to review it?

Reply:
Checking with them now!

Reply:
Here we go:

> Enabling this option will allow other AO3 users to invite your work into their collection. Your work won't be added to the collection until you have accepted the request. If you have this option enabled, you can find any collection invitations in your Manage Collected Works page. You will also receive an email notifying you of the request.
>
> Leaving this option disabled will prevent other users from inviting your works into their collections at all, and you won't receive any notifications.
>
> Changing this setting won't affect any existing works in collections.

If there are any inaccuracies in that where the changes may have deviated from the Jira issue, please lmk and I'll take it back to Docs for edits.

Reply:
...instead, Docs has edited their own text. 😆 Sorry about that. This should be the text:

>Enabling this option will allow other AO3 users to invite your work into their collection. Your work won't be added to the collection until you have accepted the request. For more information on accepting collection invitations, refer to <a href="/faq/collections#collectionitems>How do I approve or reject an invitation to include my work in a collection?</a>
>
>Leaving this option disabled will prevent other users from inviting your works into their collections at all, and you won't receive any notifications.
>
>Changing this setting won't affect any existing works in collections.

Reply:
No worries! I did my version as a placeholder so I didn't forget to ask you folks. Here's what that text looks on my local setup; does anything seem out of place here?

![updated-collections-preferences-docs](https://user-images.githubusercontent.com/13002992/205191124-dcb4ceec-542b-4441-95e0-7a756ccaa3a1.png)

Reply:
I think it needs paragraph tags, since Docs intended it to be three separate paragraphs, as it is above.

Reply:
Sure thing! In terms of appearance, do you have a preference for either of these?

Extra line breaks:
![Collection invite settings help with line breaks](https://user-images.githubusercontent.com/13002992/206042074-8b0b6c83-bf9a-4729-84a1-84e9bf5b92aa.png)

Just `<p>` tags
![Collection invite settings help with no line breaks](https://user-images.githubusercontent.com/13002992/206042307-2b073825-d4e3-4b8f-8178-785549d92b5f.png)

I'm leaning more towards the top one, but the line breaks seem a bit large. I suppose I could tinker with the CSS to make them smaller (or add margin between `<p>`s) if you would prefer that too.

Reply:
Just paragraph tags inside `<dd>` should be fine, similar to this existing help:

https://github.com/otwcode/otwarchive/blob/2d92ec1e100d2db8d98c791258b8ea115044975d/public/help/privacy-preferences.html#L28

We can deal with the spacing separately.